js/ui/gantt.d.ts

```diff
@@ -1417,6 +1431,34 @@ export interface dxGanttHeaderFilter {
     width?: number;
 }
 
+/**
+ * @docid
+ * @type object
+ * @namespace DevExpress.ui
+ */
+export interface dxGanttHeaderFilterSearch {
```

Comment:
Do you need to make this type visible in TypeScript? For now, it is marked as internal + deprecated in dx.all.d.ts
Consider removing "dx" prefix in its name

Reply:
Do we need to do it? I made it similar to dxGanttHeaderFilterTexts. If we need, then we should change dxGanttHeaderFilterSearch too

Reply:
We need to add new types names according to guidelines